### PR TITLE
CU-868akh0cb_update-event_sink_generator-to-match-latest-event_sink-changes

### PR DESCRIPTION
**Pull request by fiatcode-gh**

### CODE REVIEW

**Comment by da1nerd** [comment]
Task linked: [CU-868akh0cb update event_sink_generator to match latest event_sink changes](https://app.clickup.com/t/868akh0cb)

File: lib/src/event_sink_generator.dart [hunk 1]
```diff
@@ -51,6 +52,13 @@ class EventSinkGenerator extends GeneratorForAnnotation<EventSinkConfig> {
       classBuffer
           .writeln("_${event.eventPropertyName} = ${event.eventPropertyName},");
     }
+    for (var i = 0; i < remotes.length; i++) {
```
Comment (Var-J) [inline lib/src/event_sink_generator.dart]: i think its better to use for (entry in remotes) than using index loop

File: lib/src/event_sink_generator.dart [hunk 2]
```diff
@@ -96,6 +104,19 @@ class EventSinkGenerator extends GeneratorForAnnotation<EventSinkConfig> {
     }
     classBuffer.writeln('};');
 
+    // generate remote adapters map
+    classBuffer.writeln('@override');
+    classBuffer.writeln(
+        'Map<String, EventRemoteAdapter> get eventRemoteAdapters => {');
+    for (var i = 0; i < remotes.length; i++) {
```
Comment (Var-J) [inline lib/src/event_sink_generator.dart]: i think its better to use for (entry in remotes) than using index loop

**Review by Var-J — CHANGES_REQUESTED** [review]
Left some feedback

**Review by Var-J — APPROVED** [review]
LGTM

**Review by da1nerd — APPROVED** [review]

**Pull request merged**